aeon/dj_pipeline/acquisition.py

```diff
@@ -955,41 +954,56 @@ def key_source(self):
         )
 
     def make(self, key):
+
         chunk_start, chunk_end, dir_type = (Chunk & key).fetch1(
             "chunk_start", "chunk_end", "directory_type"
         )
+
+        raw_data_dir = Experiment.get_data_directory(key, directory_type=dir_type)
+
         weight_scale_description = (ExperimentWeightScale & key).fetch1(
             "weight_scale_description"
         )
 
-        raw_data_dir = Experiment.get_data_directory(key, directory_type=dir_type)
-
-        device = getattr(
+        weight_device = getattr(
```

Comment:
Can we keep the previous pattern?

```
device = getattr(
_device_schema_mapping[key["experiment_name"]], weight_scale_description
        )
```

and use `device.WeightRaw` afterward. 

There may be other data than just `WeightRaw` for the device.

Reply:
and keep the logic consistent with other `make()` functions

Reply:
ok, I'll make those changes.